syncr_backend/external_interface/drop_peer_store.py

```diff
@@ -102,10 +99,77 @@ class DropPeerStore(ABC):
     @abstractmethod
     async def request_peers(
         self, drop_id: bytes,
-    ) -> Tuple[bool, List[Tuple[str, str, str]]]:
+    ) -> Tuple[bool, List[Tuple[bytes, str, int]]]:
         pass
 
 
+class DHTPeerStore(DropPeerStore):
+    def __init__(
+        self,
+        node_id: bytes,
+        bootstrap_list: List[Tuple[str, int]],
+        listen_port: int,
+    ) -> None:
+        """
+        Sets up DHT peer store
+        :param node_id: node_id of this node
+        :param bootstrap_list: list of ip,port to bootstrap connect to dht
+        """
+        self.node_id = node_id
+        self.listen_port = listen_port
+        self.node_instance = get_dht()
```

Comment:
My understanding of the interaction here is this will raise a `TypeError` if the DHT hasn't been set up yet?